jobs/evidence_script_cel.go

```diff
@@ -0,0 +1,68 @@
+package jobs
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/flanksource/incident-commander/api"
+	"github.com/flanksource/incident-commander/db"
+
+	"github.com/flanksource/commons/logger"
+	"github.com/google/cel-go/cel"
+)
+
+func EvaluateEvidenceScripts() {
+	logger.Debugf("Evaluating evidence scripts")
+
+	// Fetch all evidences of open incidents which have a script
+	evidences := db.GetEvidenceScripts()
+	for _, evidence := range evidences {
+		output, err := evaluate(evidence)
+		if err != nil {
+			logger.Errorf("Error running evidence script: %v", err)
+			if err = db.UpdateEvidenceScriptResult(evidence.ID, false, err.Error()); err != nil {
+				logger.Errorf("Error persisting evidence script result: %v", err)
+			}
+			continue
+		}
+
+		var result string
+		done, err := strconv.ParseBool(output)
+		if err != nil {
+			result = "Script should evaluate to a boolean value"
+		}
+		if err = db.UpdateEvidenceScriptResult(evidence.ID, done, result); err != nil {
+			logger.Errorf("Error persisting evidence script result: %v", err)
+		}
+	}
```

Comment:
We probably want to track the evidence items that have changed status, and then update the status accordingly:

- If all evidence items marked as the definition of done are true && the status != resolved/closed, then update the status to resolved
- If any evidence items marked as the definition of done are false && the status == resolved, then update status to the status prior to resolved e.g. investigating